testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/RemoteSshGitRepositoryTestCase.java

```diff
@@ -220,14 +222,14 @@ public void prepareTest() throws Exception {
         hostPrivateKey.flush();
         hostKeyPair.writePublicKey(publicHostKey, "");
 
-        sshServer = new SSHServer(EC_USER, Paths.get(SSH_DIR +'/' + EC_PUBKEY),
+        sshServer = new SSHServer(EC_USER, Paths.get(SSH_DIR + '/' + EC_PUBKEY),
```

Comment:
@ehsavoie the tests pass so it might be a silly question but should we use proper File constant for the file separator to make sure it works on Unix and Windows?

Reply:
Paths.get(SSH_DIR).resolve(EC_PUBKEY) would be even better i think